compatibility/test-all-installs.sh

```diff
@@ -109,9 +102,8 @@ allow_nonrelease () {
       error_echo "ERROR! \`daml build\` failed for version installed from path $tarball_path"
     fi
   else
-    cat daml_install_output
-    echo_eval check_daml_install_from_tarball_after_cache_reload $tarball_path $version_cache_behaviour
     echo_eval check_recommend_cache_reload daml_install_output
+    error_echo "ERROR: Tried to install version from tarball '$tarball_path' with cache forcibly reloaded, but \`daml install\` failed."
```

Comment:
Why do the check above if its an error path? Are we asserting that we see this specific error?
Surely if we ever come down this path, something is wrong and needs looking at. Is it to maybe make debugging later easier?

Reply:
I think we should flag that there is an issue with the error message as soon as possible, even if we're in bad path, but I can see why that would seem pointless. I'm not married to it, but it seems minor enough that I don't want to run another 2 hour compat test run for it.

I'll remove it if this current run doesn't succeed.

Reply:
Oh wait my git commit modification didn't push up so I haven't been running it anyways - I'll remove it.

Reply:
fixed